Documentation/Signoffs/Mario Kart Rev 2 - Signoff - RRSA System.md

```diff
@@ -0,0 +1,86 @@
+# Mario Kart Rev 2 - Signoff - Mario Kart Bike RRSA
+## Team 4 - Tyler Chittum ##
+
+### Function
+
+The function of the ride data acquisition system is to collect the data necessary to replay rides taken by a rider. The system is required to collect the following: speed, altitude, and video footage. The system records this data and transfers it to the ride replay system by Bluetooth when in proximity.
+
+### Constraints
+
+**Measurement accuracy:** it is imperative that the device record the altitude of the rider with a high degree of precision. If the simulated resistance is to be within 10 % of the real-world resistance as previously constrained, the combined inaccuracy of the pressure sensor, the speed sensor, and the resistance system must be below 10 %. Let the error allowance for the determination of the energy expended during exercise be 1 %. For a bike and rider of 91 kg, an altitude gain of 4.67 m results in an energy expenditure of 1 kCal since E = mgh. Accordingly, the required precision to achieve +/- 1 % accuracy at a granularity of 1 kCal is +/- 4.67 cm. The speed must also be sampled at a rate which allows reasonably fast response time. Since the use of single-magnet speed sensing is already proven as common practice for bike computers, and all potential energy calculations can be performed with only the measured altitude over time, the main constraint for speed sensing is one of durability. The reed switch used must nominally tolerate at least 1000 miles of operation. 
```

Comment:
This is a bit uncertain as to what is meant by reasonably fast response time etc. So, this is not quite precise. Though it sounds like you are on the right track.